astroquery/jplhorizons/core.py

```diff
@@ -64,11 +65,19 @@ def __init__(self, id=None, location=None, epochs=None,
             element and vector queries. If no epochs are provided, the current
             time is used.
         id_type : str, optional
-            Identifier type, options: ``'smallbody'``, ``'majorbody'`` (planets
-            but also anything that is not a small body), ``'designation'``,
-            ``'name'``, ``'asteroid_name'``, ``'comet_name'``, ``'id'``
-            (Horizons id number), or ``'smallbody'`` (find the closest match
-            under any id_type), default: ``'smallbody'``
+            Controls Horizons's object selection for ``id``
+            [HORIZONSDOC_SELECTION]_ .  Options: ``'majorbody'`` (DEPRECATED,
```

Comment:
we tend to remove the deprecated functionalities from docs at the time of deprecation, so maybe just list the still valid option in the docsring.

Reply:
Done